manim/animation/animation.py

```diff
@@ -26,28 +26,28 @@
 class Animation:
     def __init__(
         self,
-        mobject: Mobject,
+        mobject: Optional[Mobject],
```

Comment:
maybe it will make more sense I think
```suggestion
        mobject: Optional[Mobject] = None,
```

Reply:
why? I don't understand how this makes more sense. I'm confused.

I would not set the default mobject to None. In fact in only recently got added in the special case of the Wait animation.

Reply:
I think `Optional` is used when the default value is `None`, and it seems it doesn't hurt here to add that right?

Reply:
Optional in this case is used as a synonym for `Union[mobject, None]`

Reply:
It's a question of intent. None is an exception, a mobject is the rule.
I'd prefer to rewrite `Optional[mobject]` as the equivalent `Union[mobject, None]` and not set a default. It just send the wrong message I think.

Reply:
I have changed it so that the self.mobject is no longer optional.